src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -57,11 +57,13 @@ object TezosNodeOperator {
   */
 class TezosNodeOperator(val node: TezosRPCInterface, val network: String, batchConf: BatchFetchConfiguration)(implicit val fetchFutureContext: ExecutionContext)
   extends LazyLogging
-  with BlocksDataFetchers {
+  with BlocksDataFetchers
+  with AccountsDataFetchers {
   import TezosNodeOperator.isGenesis
   import batchConf.{accountConcurrencyLevel, blockOperationsConcurrencyLevel, blockPageSize}
 
   override val fetchConcurrency = blockOperationsConcurrencyLevel
+  override val accountsFetchConcurrency = accountConcurrencyLevel
```

Comment:
how do we tune concurrency levels for blocks vs accounts again?

Reply:
To be honest this difference is so subtle that I don't even know anymore if the different tuning is worth it.

The parallelism involved is _only_ used when the http responses are streamed back and the utf8 body content is extracted eagerly.
The processing is quite minimal at this point, and I'm not even sure if the concurrency is getting us any real gain.

But since we could change things later to do more processing in parallel, I'd leave it as is for the moment